src/notte/env.py

```diff
@@ -130,17 +131,45 @@ def enable_auto_scrape(self: Self) -> Self:
         return self._copy_and_validate(auto_scrape=True)
 
     def use_llm(self: Self) -> Self:
-        return self.llm_data_extract().llm_action_tagging()
+        return self._copy_and_validate(
+            action=self.action.set_llm_tagging(),
+            scraping=self.scraping.llm_extract(),
+        )
 
     def disable_perception(self: Self) -> Self:
-        return self._copy_and_validate(
-            scraping=self.scraping.set_simple(),
-            action=self.action.set_simple(),
-        ).disable_auto_scrape()
+        return self._copy_and_validate(perception_model=None)
```

Comment:
that's not correct, you should keep the existing code here. We need to set 
```
scraping=self.scraping.set_simple(),
action=self.action.set_simple(),
```
But i's good to add `perception_model=None` as an extra check